queries.sql

```diff
@@ -1,3 +1,72 @@
-/*Queries that provide answers to the questions from all projects.*/
+-- Finding all the animals which ends with mon.
+SELECT * from animals WHERE name like '%mon';
+-- List the name of all animals born between 2016 and 2019.
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+-- List name and escape attempts of animals that weigh more than 10.5kg
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+-- Find all animals that are neutered.
+SELECT * FROM animals WHERE neutered = true ;
+-- Find all animals not named Gabumon.
+SELECT * FROM animals WHERE NOT name = 'Gabumon';
+-- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
 
-SELECT * from animals WHERE name = 'Luna';
+-- DAY 2
+-- Update the animals table by setting the species column to unspecified.
+-- Verify that change was made. Then roll back the change
+-- Verify that the species columns went back to the state before the transaction.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+TABLE animals;
+ROLLBACK;
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+-- Commit the transaction.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+TABLE animals;
+COMMIT;
```

Comment:
- [ ] Please use the `TABLE animals` query after `COMIT` to verify that change was undone :slightly_smiling_face:

Reply:
- [ ]  In accordance to the project requirement, we are to verify if the changes persist after "`SET species`" command, to achieve this I think you can use the select query to check as suggested earlier. 🙏

<img width="715" alt="Untitled" src="https://github.com/Kingno3l/Vet-Clinic/assets/103386183/1314142a-416d-4656-8659-b25aa8a059ac">